content/en/blog/getting-started/index.md

```diff
@@ -0,0 +1,252 @@
+exposed to the script environment so that further CLI calls can utilize it
+automatically.
+
+For the agent to issue credentials, an applicable schema needs to exist.
+The schema describes the contents of a credential, i.e., which attributes the credential contains.
+The sample script creates a schema "foobar" with a single attribute "foo" (**4-5**).
+
+There needs to be more than the mere schema for the issuing process; the agent needs also to create
+and publish its credential definition (**6-7**) attached to the created schema so that it can issue
+credentials and verifiers can verify the proof presentations against the published credential definition.
+
+We assume that the holder operates a web wallet and has taken it into use (**8-9**).
+
+```mermaid
+sequenceDiagram
+  autonumber
+    participant CLI
+    participant Agency
+    participant Web Wallet
+
+    CLI->>Agency: Register
+    CLI->>Agency: Login
+    Agency-->>CLI: JWT token
+    CLI->>Agency: Create schema
+    Agency-->>CLI: Schema id
+    CLI->>Agency: Create cred def
+    Agency-->>CLI: Cred def id
+    Note left of Web Wallet: Holder takes the web wallet in use
+    Web Wallet->>Agency: Register
+    Web Wallet->>Agency: Login
+```
+
+### CLI Script Issuing Credential
+
+The next task is to create a pairwise connection between the agent operated by the CLI user
+and the web wallet user. The pairwise connection is an encrypted pipe between the two
+agents that they can use to exchange data securely. The CLI script creates an invitation (**1-2**)
+and prints it out (**3**) as a QR code that the web wallet user can read (**5**).
+
+Once the new connection id is known, the CLI script starts a chatbot (**4**) for the new connection.
+The bot logic follows the rules for changing the bot states in the YAML configuration file. Therefore,
+the bot handles the rest of the issuing process (**6-7**).
+
+Once the issuer bot notices that credential issuing succeeded, it stops the bot (**10-11**), and
+the sample script moves on to verifying the same credential.
+
+```mermaid
+sequenceDiagram
+  autonumber
+    participant CLI
+    participant Issue Bot
+    participant Agency
+    participant Web Wallet
+
+    CLI->>Agency: Create invitation
+    Agency-->>CLI: Invitation URL
+    CLI-->>Web Wallet: <<show QR code>
+    CLI->>Issue Bot: Start
+    Web Wallet->>Agency: Read QR code
+    Agency-->>Issue Bot: Connection ready!
+    Issue Bot->>Agency: Issue credential
+    Agency-->>Web Wallet: Accept offer?
+    Web Wallet->>Agency: Accept
+    Agency-->>Issue Bot: Issue ready!
+    Issue Bot->>Issue Bot: Terminate
+```
+
+### CLI Script Verifying Credential
+
+Steps **1-6** proceed similarly to the issuing: first, the agents form a new pairwise connection.
+However, the process continues with a proof request sent by the verifier bot (**7**). The proof request
+contains the attributes the bot wishes the holder to reveal. The web wallet user sees
```

Comment:
reveal -> include in a proof presentation. 

Yes, this is hairsplitting, but if we want to keep ZKP in the game, that's more correct than 'revealing' the attributes, even when that's precisely what we are currently doing in most cases. We aren't using predicates in proof requests, etc.

Reply:
To present